src/components/NavBar/SearchBar.css.ts

```diff
@@ -142,11 +141,12 @@ export const searchBarDropdownNft = style([
   sprinkles({
     borderBottomLeftRadius: '12',
     borderBottomRightRadius: '12',
-    background: 'backgroundSurface',
     height: { sm: 'viewHeight', md: 'auto' },
   }),
   {
+    backdropFilter: 'blur(60px)',
     borderTop: 'none',
+    backgroundColor: vars.color.backgroundSurface,
```

Comment:
this should still be located in the sprinkles section of this styles as `backgroundColor: 'backgroundSurface'`